.travis.yml

```diff
@@ -13,7 +13,7 @@ script:
   # This script should be the first that runs to reduce the risk of
   # executing a script from a compromised NPM package.
   - audit-ci --moderate
-  - gulp build
-  - gulp dev &
+  - yarn run build
+  - yarn run watch &
```

Comment:
Hmm, why not just use `gulp` directly?

Reply:
Just in case there is no global gulp instance setup or global gulp version does not meet our need. Actually, we should encourage people using `yarn run` or `npm run` as it will guarantee picking up the right one(./node_modules/.bin/gulp) that we specify in package.json file.

Reply:
Ah, interesting. I didn't realize it could execute tasks without `gulp-cli` if you're using `yarn run`. If you don't need a global `gulp-cli` at all, that's a good deal.

Note that there is no global `gulp`, only global `gulp-cli`, which knows how to run the local `gulp`.

Reply:
Good point. I'll give it a try.

Reply:
It works. I removed installing global gulp-cli